zng/type.go

```diff
@@ -108,6 +109,7 @@ const (
 	TypeDefRecord = 0x80
 	TypeDefArray  = 0x81
 	TypeDefSet    = 0x82
+	TypeDefUnion  = 0x84
 	TypeDefAlias  = 0x83
```

Comment:
Nit: Maybe keep these sorted by value, as they currently are.

Reply:
oops yes!

Reply:
actually double-oops, the change was incorrect. per the spec, union is 83 and alias is 84.